[compiler] naive fallback to jit interpreter

wanchaol opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#37 [compiler] naive fallback to jit interpreter**

wanchaol committed:
naive fallback to jit interpreter

bwasti reviewed (changes requested):
Can you dump the exception in the warning?
Also, can you expose a "strict" mode and add that to tests?

wanchaol committed:
Update on "[compiler] naive fallback to jit interpreter"

naive fallback to jit interpreter

gh-metadata: pytorch tvm 37 gh/wanchaol/2/head

wanchaol committed:
Update on "[compiler] naive fallback to jit interpreter"

naive fallback to jit interpreter

gh-metadata: pytorch tvm 37 gh/wanchaol/2/head

wanchaol committed:
Update on "[compiler] naive fallback to jit interpreter"

naive fallback to jit interpreter

gh-metadata: pytorch tvm 37 gh/wanchaol/2/head
